### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#174] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-06 173845](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/911f256a-1644-4b2a-b8bf-9fbc19f953d3)

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione analoga a quella in classe
Definiamo $\Omega=\\{(w_1,w_2)|w_1,w_2=1,...,6\\}$ con $|\Omega|=36$ e $X=min(w_1,w_2)$:

$$X=\begin{cases}
1\ w=11,12,13,14,15,16,61,51,41,31,21\\
2\ w=22,23,24,25,26,62,52,42,32\\
3\ w=33,34,35,36,63,53,43\\
4\ w=44,45,46,64,54\\
5\ w=55,56,65\\
6\ w=66
\end{cases}
$$

Sia $X^{-1}(x)=\\{(w_1,w_2)| min(w_1,w_2)=x\\}$
Siccome su $\Omega$ la probabilità è uniforme $\mathbb{P}(X=x)=|X^{-1}(x)|/|\Omega|=[2(7-x)-1]/36$

<p align="center">
<img width=300 src=https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/4e5c1594-8953-43ce-b668-54c87b14c80f)
</p>

$$\mathbb{E}(X)=\sum^6_{x=1}x\cdot [2(7-x)-1]/36=\frac{1}{36}\sum^6_{x=1}(13x-2x^2)=\frac{1}{36}(13\cdot \frac{6\cdot 7}{2}-2\cdot \frac{6\cdot 7\cdot 13}{6})=\frac{91}{36}\simeq 2.53$$

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli 
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/bc6cc84e-f46f-43eb-b3d3-835755f3899d)

**Comment by fraxb04** [comment]
> Analogo @Elia-Belli ![image](https://private-user-images.githubusercontent.com/79638284/282297217-bc6cc84e-f46f-43eb-b3d3-835755f3899d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0NjQ1NDAsIm5iZiI6MTcwMjQ2NDI0MCwicGF0aCI6Ii83OTYzODI4NC8yODIyOTcyMTctYmM2Y2M4NGUtZjQ2Zi00M2ViLWIzZDMtODM1NzU1ZjM4OTlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEzVDEwNDQwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4NjhjZTcwZTdlNThlYTFlYmIxZjQ1OWFiMzg0Mzg1MTE1MWVhMmQ2NzA4YzdiMzBkZTNiYzRmZTI3MjM0YTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.31ntZcy4PXhWZyz0lQ9cCVDAMk79gFm1u0e4JDa-oz4)

non riesco a capire P(X = x) come lo hai scritto, o meglio non capisco come hai costruito quell'equazione (alla fine del punto 1), per il resto tutto corretto

**Reply by niccolo-fato** [reply]
![IMAGE 2024-02-17 15:32:42](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/73995334/d0439328-72bc-4259-a544-caa7f72524e3)
